protocol/encryption/protocol.go

```diff
@@ -234,24 +234,52 @@ func (p *Protocol) GenerateHashRatchetKey(groupID []byte) (*HashRatchetKeyCompat
 	return p.encryptor.GenerateHashRatchetKey(groupID)
 }
 
-func (p *Protocol) GetAllHREncodedKeys(groupID []byte) ([]byte, error) {
-	keys, err := p.encryptor.persistence.GetKeysForGroup(groupID)
+func (p *Protocol) GetAllHRKeysMarshaledDeprecated(groupID []byte) ([]byte, error) {
```

Comment:
Same here, maybe marge the function as Deprecated instead? And name the new function with V2 or smth.